templates/availability/_pagination.html.twig

```diff
@@ -0,0 +1,17 @@
+{% set nowIsCurrent = (form.vars.value.availabilityDomains[0].date|date('o-\\WW')) == ('now'|date('o-\\WW')) %}
+<nav>
+    <ul class="pagination">
+        <li class="page-item{% if nowIsCurrent %} active{% endif %}">
+            <a class="page-link" href="{{ path('user_availability', { week: 'now'|date('o-\\WW') }) }}">
```

Comment:
this file is also used for commissionable assets availability, it's not the same path : http://resop.vcap.me:7500/organizations/commissionable-assets/3/availability

Reply:
Can you disable the current page ?